### PR TITLE
glib: update to 2.60.0.

### PR DESCRIPTION
**Pull request by Cogitri**
We can't update yet because glib's static libs are broken,
which breaks qemu-user-static. See https://github.com/mesonbuild/meson/pull/3939

### CODE REVIEW

**Comment by atweiden** [comment]
[n1](https://gitlab.gnome.org/GNOME/glib/issues/1526)

**Comment by Cogitri** [comment]
How about we just make a libglib-static-devel package for now? Not sure how long meson upstream will take to fix this.

**Comment by Cogitri** [comment]
Alright, have added libglib-static-devel, please take another look at this @maxice8

File: srcpkgs/libglib-static-devel/template [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Template file for 'libglib-static-devel'
+#
+# This is only used for qemu-user-static until meson figures out building static
+# glib 2.60.0 via meson.
+#
+pkgname=libglib-static-devel
```
Comment (maxice8) [inline srcpkgs/libglib-static-devel/template]: have it be libglib-static

File: srcpkgs/qemu-user-static/template [hunk 2]
```diff
@@ -77,7 +77,7 @@ for _fmt in $_fmts; do
 	fi
 done
 
-binfmts="${binfmts%?}"
+binfmts="$binfmts%?"
```
Comment (maxice8) [inline srcpkgs/qemu-user-static/template]: this is a fuckup of template-formalize please run with NO_FORMALIZE=1

File: srcpkgs/libglib-static-devel/template [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+short_desc="GNU library of C routines"
+maintainer="Rasmus Thomsen <oss@cogitri.dev>"
+license="LGPL-2.1-or-later"
+homepage="https://wiki.gnome.org/Projects/GLib"
+changelog="https://gitlab.gnome.org/GNOME/glib/raw/master/NEWS"
+distfiles="${GNOME_SITE}/glib/${version%.*}/glib-${version}.tar.xz"
+checksum=8f43c31767e88a25da72b52a40f3301fefc49a665b56dc10ee7cc9565cbe7481
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" glib-devel"
```
Comment (maxice8) [inline srcpkgs/libglib-static-devel/template]: needs to change here as well no ?

Comment (Cogitri) [reply srcpkgs/libglib-static-devel/template]: No, it still needs the glib binaries for cross building

**Pull request merged**